torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1432,6 +1433,14 @@ XLATensor XLATensor::softmax(const XLATensor& input, xla::int64 dim) {
       XlaHelpers::GetCanonicalDimensionIndex(dim, input.shape().get().rank())));
 }
 
+XLATensor XLATensor::softmax_backward(const XLATensor& grad_output,
+                                      const XLATensor& output, xla::int64 dim) {
+  return grad_output.CreateFrom(ir::MakeNode<ir::ops::SoftmaxBackward>(
```

Comment:
This is right about at the boundary, for as far as nested, single line ops, should be.

Reply:
Done.